src/jit/hwintrinsicxarch.cpp

```diff
@@ -323,6 +321,8 @@ bool Compiler::isFullyImplmentedISAClass(InstructionSet isa)
             return false;
 
         case InstructionSet_SSE:
+        case InstructionSet_SSE3:
+        case InstructionSet_SSSE3:
```

Comment:
These are fully implemented, but we actually need to condition some tests (those that use `Sse2.LoadAlignedVector128`) on whether `Sse2` is also fully implemented.